src/components/common/Modal/Modal.jsx

```diff
@@ -2,10 +2,16 @@ import PropTypes from 'prop-types';
 import s from './Modal.module.scss';
 import { Portal } from 'src/components/common';
 
-export const Modal = ({ closeModalHandler, location, children }) => (
+export const Modal = ({ closeModalHandler, location, startCloseAnimation, children }) => (
```

Comment:
move const { mounted } = useMount(isModalOpen); to this comonent, so you do not have startCloseAnimation but isModalOpen